Proposal for feature summary (Issue #159) (#160)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

key material controlled by the subject of SD-JWT. Holder binding is optional to
implement.

## Feature Summary

>  * This specification defines a format enabling selective disclosure for
>    JWTs, called SD-JWT, plus formats for associated data for disclosing
>    claims, as well as formats for the combined transport of SD-JWTs and
>    the associated data.

Reviewer: ```suggestion
 * This specification defines 
   - a format enabling selective disclosure for JWTs, called SD-JWT, 
   - formats for associated data for disclosing claims
   - formats for the combined transport of SD-JWTs and the associated data
```

>  * The format allows combining selectively disclosable claims and
>    clear-text claims that are always disclosed.
>  * The format supports flat data structures as well as more complex,
>    nested data structures.
>  * Optionally, claim names can be hidden (or "blinded").
>  * Claims with
>    blinded and unblinded names can be combined in the same SD-JWT.

Reviewer: ```suggestion
 * The specification supports selectively disclosable claims to be expressed 
    using flat data structures as well as more complex, nested data structures.
  * This specification enables combining selectively disclosable claims with
   clear-text claims that are always disclosed.
  * Optionally, this specification allows to also hide ("blind") the claim names, not only the claim values.
  * When claim names are blinded, this specification enables combining claims with 
     blinded and unblinded names in the same SD-JWT.

```

key material controlled by the subject of SD-JWT. Holder binding is optional to
implement.

## Feature Summary

>  * This specification defines 
>    - a format enabling selective disclosure for JWTs
>    - formats for associated data that enables disclosing claims
>    - formats for the combined transport of SD-JWTs and the associated data

Author: ```suggestion
 * This specification defines 
   - a format enabling selective disclosure for JWTs,
   - formats for associated data that enables disclosing claims, and
   - formats for the combined transport of SD-JWTs and the associated data.
```

>  * The specification supports selectively disclosable claims to be expressed 
>     using flat data structures as well as more complex, nested data structures.

Author: ```suggestion
 * The specification supports selectively disclosable claims in flat data structures as well as more complex, nested data structures.
```
